### PR TITLE
mixbox 1.0.5 :snowflake:

### PR DESCRIPTION
**Pull request by lorepirri**
mixbox 1.0.5 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4170]
- dev_url (pypi): https://github.com/CybOXProject/mixbox/tree/v1.0.5  
- conda-forge: None
- pypi: [https://pypi.org/project/mixbox/](https://pypi.org/project/mixbox/#files)
- inspector: https://inspector.pypi.io/project/mixbox/1.0.5/

### Depends on

- AnacondaRecipes/weakrefmethod-feedstock#1

### Explanation of changes:

- new feedstock, add SF channel
- unvendor `six`, use `six >=1.16.0`, because `six 1.15.0` has this problem: https://github.com/benjaminp/six/issues/39
- add upstream tests

[PKG-4170]: https://anaconda.atlassian.net/browse/PKG-4170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by boldorider4 — APPROVED** [review]
LGTM

**Review by boldorider4 — COMMENTED** [review]
I will quickly try the upstream tests since it's the requested package.

**Comment by lorepirri** [comment]
I forgot to unvendor `six`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - lxml
+    - ordered-set
+    - python-dateutil
+    - weakrefmethod >=1.0.3
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
    - weakrefmethod >=1.0.3  # [py<34]
```
This is technically not required. They support [weakref](https://docs.python.org/3/library/weakref.html), see https://inspector.pypi.io/project/mixbox/1.0.5/packages/ff/b1/38a53aa8a059f09a06881e948cccff067c664e88f8a14cbfef8c5c29fac9/mixbox-1.0.5.tar.gz/mixbox-1.0.5/mixbox/signals.py#line.19.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - mixbox
+    - mixbox.vendor
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
    - mixbox.parser
    - mixbox.vendor
```
to test `from weakref import WeakMethod`.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -25,15 +30,21 @@ requirements:
     - ordered-set
     - python-dateutil
     - weakrefmethod >=1.0.3
+    # unvendored six
+    - six 1.15.0
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: Six is fully backward compatible. We could use a lower bound instead of a hard pin.

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok

File: recipe/meta.yaml (hunk 3 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok,

done with `six >=1.16.0`, 

six 1.15.0 has this problem: https://github.com/benjaminp/six/issues/39

**Review by JeanChristopheMorinPerso — APPROVED** [review]
Thanks for unvendoring six!

**Pull request merged**